frontend/src/components/editor/chrome/wrapper/footer.tsx

```diff
@@ -79,7 +79,13 @@ export const Footer: React.FC = () => {
       >
         {renderIcon("snippets")}
       </FooterItem>
-
+      <FooterItem
+        tooltip="Scratchpad"
+        selected={selectedPanel === "scratchpad"}
+        onClick={() => openApplication("scratchpad")}
+      >
+        {renderIcon("scratchpad")}
+      </FooterItem>
```

Comment:
we moved some footer items to a sidebar, so you will likely have a merge conflict here FYI